### PR TITLE
breaking change, setting-up routing

### LINKED ISSUE #6: setting-up routing

### PR DESCRIPTION
**Pull request by yhunlu**
>
>
> Co-authored-by: John Moonshine <web3futureofweb@gmail.com>
> Co-authored-by: Reucation <reucation@gmail.com>

### CODE REVIEW

File: src/pages/Layout.tsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import { Outlet } from "react-router-dom"
+import NavBar from "../components/NavBar"
+
+const Layout = () => {
+    return (
+        <>
+            <NavBar />
+            <Outlet />
```
Comment (mypassionscoding) [inline src/pages/Layout.tsx:8]: allowed routing, children

File: src/main.tsx [hunk 2]
```diff
@@ -16,7 +17,7 @@ ReactDOM.createRoot(document.getElementById('root')!).render(
       <SpeedInsights />
       <ColorModeScript initialColorMode={theme.config.initialColorMode} />
       <QueryClientProvider client={queryClient}>
-        <App />
+        <RouterProvider router={router} />
```
Comment (mypassionscoding) [inline src/main.tsx:20]: removed app.tsx as well as css, created homepage instead of this

**Review by mypassionscoding — COMMENTED** [review]
+

**Review by yhunlu — COMMENTED** [review]
+

File: src/main.tsx (hunk 2 above)
Comment (JohnMoonshine) [reply src/main.tsx:20]: oh, that can break application!, thanks informing

**Review by JohnMoonshine — COMMENTED** [review]
+

File: src/pages/HomePage.tsx
Comment (reucation) [inline src/pages/HomePage.tsx:1]: HomePage instead of App?, okay..

**Review by reucation — COMMENTED** [review]
+

**Pull request merged**